.gitignore

```diff
@@ -30,19 +30,21 @@ yarn-error.log*
 /kumascript/coverage/
 /kumascript/node_modules/
 /testing/node_modules/
-/build/*.js
-/build/*.js.map
+/build/**/*.js
+/build/**/*.js.map
```

Comment:
(Can you update the PR title/description to account for the fact that you also update `.gitignore`, not just NPM packaging ignores?)

The single asterisk seemed to have worked, even though the [Git docs](https://git-scm.com/docs/gitignore#_pattern_format) states it shouldn't have:

> An asterisk "*" matches anything **except a slash**.

But if we fix it for `/build/`, we should also fix it for `content`, `filecheck` and `server` below.